src/components/vuestic-components/va-sidebar/VaSidebarLink.vue

```diff
@@ -87,16 +87,25 @@ export default {
 
       if (this.isHovered || this.isActive) {
         return {
-          color: this.$themes['primary'],
-          backgroundColor: getBackgroundColor(),
-          borderColor: this.isActive ? this.$themes['primary'] : 'transparent',
+          color: this.isDefaultColorTheme ? this.$themes.primary : 'white',
```

Comment:
That is not ideal way to handle this hack.

More correct way would be to do a computed, like `noGradient`, `invertFontColor`, `shadowType`. As that would be consistent with what we plan to achieve with context.

Please don't fix. Just for your information.

Reply:
This component not used in vuestic-admin. May be need to remove va-sidebar components from vuestic-ui